sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -2534,9 +2534,13 @@ pi_result cuda_piextQueueCreate(pi_context Context, pi_device Device,
   assert(Properties);
   // Expect flags mask to be passed first.
   assert(Properties[0] == PI_QUEUE_FLAGS);
```

Comment:
Do you need to keep these asserts?

Reply:
I've seen the same pattern in another place in one of those. Also, I'd rather have it abort in debug to speed up any potential analysis.